src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -24,8 +24,10 @@
 using Microsoft.VisualStudio.LanguageServices.SymbolSearch;
```

Comment:
this first commit just moves them onto their async api.  teh next commit majorly cleans thigns up using new infrastructure components we have for this sort of work.